db/dcrpg/sync.go

```diff
@@ -136,8 +139,39 @@ func (db *ChainDB) SyncChainDB(client *rpcclient.Client, quit chan struct{},
 			return ib - 1, fmt.Errorf("GetBlock failed (%s): %v", blockHash, err)
 		}
 
+		var winners []string
+		//prevBlockHash := block.MsgBlock().Header.PrevBlock
+		for {
```

Comment:
A little wait loop to facilitate the simultaneous sync of sqlite (which manages the shared stakedb) and postgresql.